src/cgame/cg_main.cpp

```diff
@@ -1337,6 +1319,9 @@ static void CG_RegisterClients()
 		CG_NewClientInfo( i );
```

Comment:
I think we eagerly load everything before this so it doesn't actually load anything new.